crates/umem_grpc_server/proto/memory.proto

```
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package memory;
```

Buf (1.54.0)

Files with package "memory" must be within a directory "memory" relative to root but were in directory "crates/umem_grpc_server/proto".

(PACKAGE_DIRECTORY_MATCH)